Apps/Medication/test/unit/Delegates/SalesforceDelegate_Test.cs

```diff
@@ -0,0 +1,346 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Medication.Delegates.Test
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Net;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using System.Threading;
+
+    using HealthGateway.Common.AccessManagement.Authentication.Models;
+    using HealthGateway.Common.AccessManagement.Authentication;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Common.Services;
+    using HealthGateway.Medication.Models;
+
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+
+    using Moq.Protected;
+    using Moq;
+
+    using Xunit;
+
+    public class SalesforceDelegate_Test
+    {
+        [Fact]
+        public void ShouldRetrieveMedicationRequests()
+        {
+            // Setup
+
+            // Input Parameters
+            string phn = "9735361219";
+
+            // Setup Configuration
+            string endpoint = "https://test-endpoint";
+            Uri tokenUri = new Uri("https://localhost");
+            ClientCredentialsTokenRequest tokenRequest = new ClientCredentialsTokenRequest()
+            {
+                ClientId = "TEST_CLIENTID",
+                ClientSecret = "TEST_CLIENT_SECRET",
+                Password = "TEST_PASSWORD",
+                Username = "TEST_USERNAME",
+            };
+            Dictionary<string, string> configurationParams = new Dictionary<string, string>
+            {
+                { "Salesforce:Endpoint", endpoint },
+                { "Salesforce:TokenUri", tokenUri.ToString() },
+                { "Salesforce:ClientAuthentication:ClientId", tokenRequest.ClientId },
+                { "Salesforce:ClientAuthentication:ClientSecret", tokenRequest.ClientSecret },
+                { "Salesforce:ClientAuthentication:Username", tokenRequest.Username },
+                {"Salesforce:ClientAuthentication:Password", tokenRequest.Password },
```

Comment:
Formatting - missing space.

Reply:
Also applies to similar sections in this file.

Reply:
fixed